### PR TITLE
add Contour conformance reports for v1.27.1, v1.27.2, v1.28.3

### PR DESCRIPTION
**Pull request by skriss**
**What type of PR is this?**
/area conformance

**What this PR does / why we need it**:
Adds conformance reports and implementation page update for Contour v1.27.1, v1.27.2, and v1.28.3

**Which issue(s) this PR fixes**:
N/A

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by skriss** [comment]
cc @sunjayBhatia

**Review by sunjayBhatia — APPROVED** [review]

File: conformance/reports/v0.8.1/projectcontour-contour/README.md [hunk 1]
```diff
@@ -5,5 +5,7 @@
 |API channel|Implementation version|Mode|Report|
 |-----------|----------------------|----|------|
 |x|[v1.27.0](https://github.com/projectcontour/contour/releases/tag/v1.27.0)|x|[link](./v1.27.0-report.yaml)|
+|x|[v1.27.1](https://github.com/projectcontour/contour/releases/tag/v1.27.1)|x|[link](./v1.27.1-report.yaml)|
+|x|[v1.27.2](https://github.com/projectcontour/contour/releases/tag/v1.27.2)|x|[link](./v1.27.2-report.yaml)|
 
 ## Reproduce
```
Comment (robscott) [inline conformance/reports/v0.8.1/projectcontour-contour/README.md:11]: We've started to require reproduction steps for new conformance reports. Mind adding something like https://github.com/kubernetes-sigs/gateway-api/tree/main/conformance/reports/v1.0.0/kong-kubernetes-ingress-controller here?

**Review by robscott — COMMENTED** [review]
Thanks @skriss!

File: conformance/reports/v0.8.1/projectcontour-contour/README.md (hunk 1 above)
Comment (robscott) [reply conformance/reports/v0.8.1/projectcontour-contour/README.md:11]: 🤦  just noticed this is v0.8 and you already have reproduction steps for v1.0, closing this out.

**Comment by robscott** [comment]
Not sure why Dave's LGTM didn't appear to work.

/lgtm

**Pull request merged**